### ISSUE TITLE
Incremental Release V05-01-04 (Trac #2037)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2029]
The latest filterscript release has a bug that let the pass2 and 2017 scripts fail.

I think we need a new filterscripts release that includes r156184 (http://code.icecube.wisc.edu/projects/icecube/changeset/156184/IceCube) and then this incremental metaproject release.

Migrated from https://code.icecube.wisc.edu/ticket/2037
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:57", 
    "description": "The latest filterscript release has a bug that let the pass2 and 2017 scripts fail.\n\nI think we need a new filterscripts release that includes r156184 (http://code.icecube.wisc.edu/projects/icecube/changeset/156184/IceCube) and then this incremental metaproject release.", 
    "reporter": "joertlin", 
    "cc": "blaufuss", 
    "resolution": "fixed", 
    "_ts": "1550067237750774", 
    "component": "combo reconstruction", 
    "summary": "Incremental Release V05-01-04", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2017-05-31T21:21:38", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170601T16:04:38: 
* olivas changed owner from "kjmeagher" to "olivas"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20170601T18:01:21: 
* olivas changed status from "assigned" to "closed"
* olivas changed resolution from "" to "fixed"
* olivas commented:

> In changeset 156240:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="156240"
> updated release notes and externals to include latest filterscripts.  closes #2029.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:57: 
* olivas commented:

> Milestone June 2017 deleted


* olivas changed milestone from "June 2017" to ""
